vermouth/tests/pdb/test_read_pdb.py

```diff
@@ -182,3 +183,52 @@ def test_altloc(caplog):
     mol = mols[0]
     assert len(mol.nodes) == 1
     assert any(rec.levelname == 'WARNING' for rec in caplog.records)
+
+
+def test_atom_attributes():
+    """
+    Test that atom attributes are parsed and set correctly
+    """
+    pdb = multi_mol(False)
+    parser = PDBParser()
+    mols = list(parser.parse(pdb.splitlines()))
+    assert len(mols) == 2
+    nodes = [
+        {
+            0: {'atomname': 'C01', 'atomid': 1, 'resid': 1,
+                'resname': 'UNK', 'chain':'', 'altloc': '',
+                'insertion_code': '', 'occupancy': 0.0, 'temp_factor': 0.0,
+                'position': np.array([0.3939,  -0.2095,  -0.4491]),
+                'charge': 2.0, 'element': 'C'},
+            1: {'atomname': 'C02', 'atomid': 2, 'resid': 1,
+                'resname': 'UNK', 'chain':'', 'altloc': '',
+                'insertion_code': '', 'occupancy': 0.0, 'temp_factor': 0.0,
+                'position': np.array([0.4825,  -0.2409,  -0.5534]),
+                'charge': 0, 'element': 'C'}
+        },
+        {
+            0: {'atomname': 'C03', 'atomid': 4, 'resid': 1,
+                'resname': 'UNK', 'chain': '', 'altloc': '',
+                'insertion_code': '', 'occupancy': 0.0, 'temp_factor': 0.0,
+                'position': np.array([0.5466,  -0.3658,  -0.5560]),
+                'charge': 0, 'element': 'C'},
+            1: {'atomname': 'C04', 'atomid': 5, 'resid': 1,
+                'resname': 'UNK', 'chain': '', 'altloc': '',
+                'insertion_code': '', 'occupancy': 0.0, 'temp_factor': 0.0,
+                'position': np.array([0.5207,  -0.4601,  -0.4552]),
+                'charge': 0, 'element': 'C'},
+            2: {'atomname': 'C05', 'atomid': 6, 'resid': 1,
+                'resname': 'UNK', 'chain': '', 'altloc': '',
+                'insertion_code': '', 'occupancy': 0.0, 'temp_factor': 0.0,
+                'position': np.array([0.5207, -0.4601, -0.4552]),
+                'charge': 0, 'element': 'C'}
+        }
+    ]
+    for mol, n_attrs in zip(mols, nodes):
```

Comment:
Can you be sure the order of moles is always the same as that of nodes?

Reply:
I think so. The first molecule in the PDB should also always be the first molecule in the system.